dipy/workflows/align.py

```diff
@@ -4,17 +4,26 @@
 from dipy.io.image import load_nifti, save_nifti
 from dipy.workflows.workflow import Workflow
 
+import numpy as np
+import nibabel as nib
+from dipy.align.imaffine import (transform_centers_of_mass, AffineMap,
+                                 MutualInformationMetric, AffineRegistration)
+from dipy.align.transforms import (TranslationTransform3D, RigidTransform3D,
+                                   AffineTransform3D)
+from dipy.io.image import save_nifti, save_affine_matrix, \
+    save_quality_assur_metric
```

Comment:
I don't know if DiPy has a standard or not, but since it's your own code, can you stick to one way? Either `import ()` or `import \`, but not both.

Reply:
Done, Thanks. Actually, part of this file was contributed was someone else earlier and so I did not fiddle with the parts that were already there but you have a valid point, I also support code consistency and standard. It's better to just stick to a single standard rather than using different ones. Good catch, I did not even notice it until you pointed this out.